node/src/components/consensus.rs

```diff
@@ -357,6 +363,29 @@ where
                 let validator_changes = self.get_validator_changes();
                 responder.respond(validator_changes).ignore()
             }
+            Event::DumpState(req @ DumpConsensusStateRequest { era_id, .. }) => {
```

Comment:
Do we need to bind the `req @` if it's not used later? Wouldn't
```suggestion
            Event::DumpState(DumpConsensusStateRequest { era_id, .. }) => {
```
suffice?

Reply:
Oh wait, there is `req` used down below 🙈